Sources/StreamVideoSwiftUI/CallView/CornerDraggableView.swift

```diff
@@ -45,14 +52,7 @@ public struct CornerDraggableView<Content: View>: View {
     public var body: some View {
         content(availableFrame)
             .overlay(
-                // SwiftUI seems to remove any interaction of views that not
```

Comment:
why do we remove these? There's some useful info here.

Reply:
My mistake. I brought them back